docs/source/conf.py

```diff
@@ -1467,6 +1467,8 @@
     "stft",
     # torch.onnx.symbolic_opset18
     "col2im",
+    # torch.onnx.symbolic_opset20
+    "affine_grid_generator",
```

Comment:
Please do not add new entries here.
Make sure that either this function is properly documented on the docs website or make it private by adding a `_` at the beginning of the name